justfile

```diff
@@ -103,7 +103,7 @@ doc:
     set -e
     cargo doc
 
-lint: clean
+lint:
```

Comment:
This commit removes clean from `just lint` but the commit message says it removes it from `just build`

Reply:
fixed